redshift_connector/cursor.py

```diff
@@ -377,11 +404,13 @@ def fetch_dataframe(self: "Cursor", num: typing.Optional[int] = None) -> typing.
         try:
             import pandas
         except ModuleNotFoundError:
-            raise ModuleNotFoundError(MISSING_MODULE_ERROR_MSG.format(module="pandas"))
+            raise ModuleNotFoundError(
+                MISSING_MODULE_ERROR_MSG.format(module="pandas"))
 
         columns: typing.Optional[typing.List[typing.Union[str, bytes]]] = None
         try:
-            columns = [column[0].decode().lower() for column in self.description]
+            columns = [column[0].decode().lower()
+                       for column in self.description]
```

Comment:
@Yash621 is this necessary? adds noise to the PR.

Reply:
fixed in revisions